src/test/librbd/test_internal.cc

```diff
@@ -538,15 +538,20 @@ TEST_F(TestInternal, Metadata) {
   map<string, bufferlist> pairs;
   r = librbd::metadata_list(ictx, "", 0, &pairs);
   ASSERT_EQ(0, r);
-  ASSERT_EQ(5u, pairs.size());
+
+  uint8_t original_paris_num = 0;
```

Comment:
Is this just a typo?

```suggestion
  uint8_t original_pairs_count = 0;
```

Reply:
Updated